src/cobalt_model/site.rs

```diff
@@ -73,6 +77,11 @@ impl Site {
                 liquid::model::Value::scalar(kstring::KString::from_ref(base_url)),
             );
         }
+        attributes.insert(
+            "time".into(),
```

Comment:
Could you expand in your PR on your use case for wanting this?

Reply:
Jekyll has this global variable.
I'm rewriting the areweinspaceyet.org with rust and I want to keep.
In a nutshell it shows when was the last time the website was generated (updated)

https://github.com/AeroRust/are-we-in-space-yet/blob/933b45db16c2585f68d2ce0c9a03640ad6604366/_includes/footer.html#L19

Reply:
Thanks!  I personally find it inaccurate, update is different than re-generate, but I can understand just wanting to carry something forward.

Could you put some of this in the PR description?

Reply:
I've added a description :+1: